app/controllers/keywords_controller.rb

```diff
@@ -10,6 +10,21 @@ def index
            })
   end
 
+  def show
+    keyword = Keyword.find(params['id'])
```

Comment:
Just a small thing we can the `:` in this case ;) 
```suggestion
    keyword = Keyword.find(params[:id])
```

Reply:
Fixed in e94cb62675e0fb97cad1c9873885989bc859daa4.

Small question though. What is your reason the favor one over another?

As I understand, Ruby symbols bear the same semantic with Elixir atoms. They both indicates something known to your system. You can argue that your system should know about the `:id` parameter since it comes from Rails router path. So in this case, using symbol is ok.

I wonder about the parameters which users supply:
```
# Should we use string
params['some_wild_parameter']

# Or use symbol here?
params['some_wild_parameter']
```

Reply:
@Goose97 First let talk about the Rails Parameter; it will convert the Hash params to the symbol key Hash, so either it's the `param from the Rails route` or from the `user supply - through a form` - Rails parameter will convert that, and in the controller, we always can use the `symbol` way -> `params[:some_wild_parameter]`

Back to the reason why we prefer the `symbol` over `string`, first it's just shorter than the quote and makes the code look nice ;) also in Ruby by default - when it see the `string` it will init the new object in the memory and delete it later, while with `symbol` it will just create only 1 single object and keep it there.